src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBasement.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace Microsoft.Maui.DeviceTests
 {
-	public class HandlerTestBasement : TestBase, IDisposable
+	public class HandlerTestBasement : TestBase, IAsyncDisposable
```

Comment:
MAUI team reviewers: This is the part of the change that can affect other tests: I changed this to be IAsyncDisposable (which the BlazorWebView tests always did). In theory this is a good change anyway because MauiApp _is_ IAsyncDisposable so should really always be disposed that way.